### PR TITLE
Hello Microverse Project Pull Request

### PR DESCRIPTION
**Pull request by federicaulzurrun**
It´s a simple html and css project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you give your `h1` a `class name` then the class name should be used to style your h1 content in the CSS file.

E.G:
```HTML
<h1 class="header">Hello Microverse</h1>
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```
Comment (brainconnect93) [inline style.css]: - [ ] The class name that was given to your h1 in the HTML file should be used here to style the content

E.G:
``` CSS
.header {
    color: aqua;
  }
```

With this, you will be sure you follow the project requirement

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @federicaulzurrun, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x] Great effort working on your `Hello Microverse` project 👍 👍, and below are some of the things i want you to pay attention to:
 
    - I would kindly suggest you make use of the README file template given to us by Microverse in your project and edit all the necessary headings to describe what you have in your project. Also, remove the Microverse logo at the top of the README file, Optionally you can replace it with your own logo if you have one. Kindly follow this [README link](https://github.com/microverseinc/readme-template) to add the correct template to your project.

 Check the image below 👇 to know how to work around your README file
 
image

   - Also your commit messages, always make them descriptive and professional. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8).

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdallahmalima — APPROVED** [review]
Hi @federicaulzurrun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - Professional README File ✔ 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**